sycl/tools/sycl-ls/sycl-ls.cpp

```diff
@@ -68,6 +68,106 @@ std::string getDeviceTypeName(const device &Device) {
   }
 }
 
+namespace syclex = sycl::ext::oneapi::experimental;
```

Comment:
This has to be at least local to the `getArchName` function, if not fully inlined.